helm/chaos-mesh/templates/controller-manager-rbac.yaml

```diff
@@ -14,9 +14,9 @@ metadata:
 ---
 # roles
 kind: ClusterRole
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Why does this PR modify this line?

If you need, please cherry-pick #994

Reply:
Okay, I have cerated this pr to cherry-pick it https://github.com/chaos-mesh/chaos-mesh/pull/1296